### PR TITLE
Feature user profile

### PR DESCRIPTION
**Pull request by Ashwin-op**

### CODE REVIEW

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <input (ngModelChange)="searchQueryChanged.next($event)" [ngModel]="searchQuery" nbInput
+             placeholder="Search events, labs, communities, people" type="text">
+    </nb-form-field>
+    <nb-select (selectedChange)="getSelectedFilters($event)" class="search-filter" multiple nbSuffix
+               placeholder="Filters">
+      <nb-option *ngFor="let filter of filterBy" [value]="filter">{{filter | titlecase}}</nb-option>
+    </nb-select>
+  </div>
+
+  <div class="home-navbar-search-suffix">
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.html]: we can just shorten the class to suffix or search suffix

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <div [ngClass]="{'active': showSearchResults}" class="home-navbar-search-results d-flex flex-column">
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.html]: shorten classname

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <div *ngFor="let result of searchResults | keyvalue">
+        <div *ngIf="result.value.length !== 0" class="home-navbar-search-result">
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.html]: same!

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          <div class="search-result-heading">{{result.key}}</div>
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.html:18]: results-group

File: projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-content/user-contributions/user-contributions.component.scss [hunk 5]
```diff
@@ -1,43 +1,62 @@
...
+      line-height: 28px;
+      color: $neon-blue;
     }
-  }
 
-  .user-contributions-labs {
-    .user-contributions-labs-cards {
-      overflow: auto;
+    .content {
+      display: flex;
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-content/user-contributions/user-contributions.component.scss:16]: should we move this to classes?

**Review by arpansac — CHANGES_REQUESTED** [review]

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html (hunk 1 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/components/search-bar/search-bar.component.html]: Done!

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html (hunk 2 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/components/search-bar/search-bar.component.html]: Done!

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html (hunk 3 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/components/search-bar/search-bar.component.html]: Done!

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html (hunk 4 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/components/search-bar/search-bar.component.html:18]: Done!

File: projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-content/user-contributions/user-contributions.component.scss (hunk 5 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-content/user-contributions/user-contributions.component.scss:16]: There are a lot of content classes and I'm selecting them as a whole, that's why I put the display in scss file so we can control them from a single place

**Pull request merged**